kartothek/utils/predicate_converter.py

```diff
@@ -0,0 +1,80 @@
+"""
+Helper module to convert kartothek dataset load predicates into cube conditions.
+"""
+
+from __future__ import absolute_import
+
+import pandas as pd
+import pyarrow as pa
+
+from kartothek.core.cube.conditions import Condition
+
+
+def write_predicate_as_cube_condition(predicate):
+    """
+    Rewrites a single io.dask.dataset 'read_dataset_as_ddf' predicate condition as cube condition
+
+    Parameters
+    ----------
+    predicate: list
+        list containing single predicate definition
+
+    Returns
+    -------
+    condition: cube condition object
+        cube condition containing the predicate definition
+    """
+    condition_string = None
+    parameter_format_dict = {}
+    if type(predicate[2]) == int:
+        condition_string = "{} {} {}".format(
+            predicate[0], predicate[1], str(predicate[2])
+        )
+        parameter_format_dict[predicate[0]] = pa.int16()
```

Comment:
Could you use `pa.types.is_int*` for this and the following?

Reply:
would that make sense in this context? It should not check the type, but rather define it for the the "to_string" conversion.  
If converting to pa.types.is_int(t), what would I need to pass here as "t"?

Reply:
Ah, sorry, I saw a `==` here. Than this is OK but I would be curious why you are defaulting to `int16` and not `int64` here?

Reply:
no particular reason for that, I saw this was used in some modules (but 32 and 64 as well). I will change it to int64